lib/diagnostic.js

```diff
@@ -46,7 +47,7 @@ let person2 = {
 };
 
 // Question 3:  Invoke the `fullName()` method so it returns `Santa Clause`?
-let answer3 = null; // replace null with your response
+let answer3 = person2.fullName(); // replace null with your response
```

Comment:
`person2` doesn't have a method called `fullName()` so this will cause an error.  You need to call `person1`'s `fullName()` method but use `Function`'s `call()` method to call it in the context of `person2`.

```javascript
person1.fullName.call(person2);
```